dotnet/src/SemanticKernel.Abstractions/SemanticFunctions/PromptTemplateConfig.cs

```diff
@@ -107,69 +54,63 @@ public class InputConfig
     }
 
     /// <summary>
-    /// Schema - Not currently used.
+    /// AI service configuration
     /// </summary>
-    [JsonPropertyName("schema")]
-    [JsonPropertyOrder(1)]
-    public int Schema { get; set; } = 1;
+    public class ServiceConfig
+    {
+        /// <summary>
+        /// Model ID in AI service provider (e.g. text-davinci-003).
+        /// </summary>
+        [JsonPropertyName("model_id")]
+        [JsonPropertyOrder(1)]
+        public string ModelId { get; set; } = string.Empty;
 
-    /// <summary>
-    /// Type, such as "completion", "embeddings", etc.
-    /// </summary>
-    /// <remarks>TODO: use enum</remarks>
-    [JsonPropertyName("type")]
-    [JsonPropertyOrder(2)]
-    public string Type { get; set; } = "completion";
+        /// <summary>
+        /// Service priority order. Kernel will check all registered services and choose one with highest priority order.
```

Comment:
what's the scenario for needing an order? Also, you can use an array to sort models, removing the need to lookup all the values and sort